eva-accession-core/src/main/java/uk/ac/ebi/eva/accession/core/ISubmittedVariant.java

```diff
@@ -41,14 +41,110 @@
 
     String getAlternateAllele();
 
-    Long getClusteredVariant();
+    Long getClusteredVariantAccession();
 
     /**
      * @return True if this submitted variant is supported by genotypes or frequencies
      */
-    boolean isSupportedByEvidence();
+    Boolean getSupportedByEvidence();
 
     Boolean getMatchesAssembly();
 
+    /**
+     * @return False if reference allele was not found in the alleles list
+     */
+    Boolean getAllelesMatch();
+
+    /**
+     * @return True if the variant was curated manually, and not only detected by computational methods.
+     * see https://www.ncbi.nlm.nih.gov/books/NBK21088/table/ch5.ch5_t4/?report=objectonly
+     */
+    Boolean getValidated();
+
     LocalDateTime getCreatedDate();
+
+    static boolean equals(ISubmittedVariant first, ISubmittedVariant second) {
```

Comment:
agreed offline to 1) extract the method to another class so it's not confused as an overriding `default equals` method and 2) remove the DTO child and add a copy constructor in SubmittedVariant so that only there an autogenerated equals is needed.